playground/src/screens/OptionsScreen.js

```diff
@@ -180,19 +205,11 @@ class OptionsScreen extends Component {
     });
   }
 
-  onClickAlert() {
-    Navigation.showOverlay('custom', 'navigation.playground.CustomDialog');
-  }
-
-  onClickSnackbar() {
-    Navigation.showOverlay('snackbar', {
-      text: 'Test!',
-      // textColor: 'red',
-      // backgroundColor: 'green',
-      duration: 'long',
-      button: {
-        text: 'Action',
-        textColor: 'blue'
+  onClickFab() {
+    Navigation.setOptions(this.props.componentId, {
+      fabMenu: {
```

Comment:
```js
Navigation.setOptions(this.props.componentId, {
  fab: {
    id: 'myFAB',
    backgroundColor: 'blue',
    actions: [
      {
         // an action which is displayed in a "menu", same api as main fab
      }
    ]
  }
});
```

This way we avoid having two entities, the actions (menu items) are a property of the fab.